lib/vsc/utils/fancylogger.py

```diff
@@ -85,6 +85,79 @@
 import weakref
 from distutils.version import LooseVersion
 
+
+def _env_to_boolean(varname, default=False):
+    """
+    Compute a boolean based on the truth value of environment variable `varname`.
+    If no variable by that name is present in `os.environ`, then return `default`.
+
+    For the purpose of this function, the string values ``'1'``,
+    ``'y'``, ``'yes'``, and ``'true'`` (case-insensitive) are all
+    mapped to the truth value ``True``::
+
+      >>> os.environ['NO_FOOBAR'] = '1'
```

Comment:
No, it doesn't... We'll need a separate unit test for it (see test/fancylogger.py)